config/config.txt

```diff
@@ -274,10 +274,10 @@ TICK_LIMIT_MC_INIT 500
 
 ##Defines the ticklag for the world. Ticklag is the amount of time between game ticks (aka byond ticks) (in 1/10ths of a second).
 ##	This also controls the client network update rate, as well as the default client fps
-#TICKLAG 0.5
+#TICKLAG 0.4
 
 ##Can also be set as per-second value, the following value is identical to the above.
-FPS 40
+FPS 60
```

Comment:
I remember people screeching about FPS and also setting the FPS serverside to twenty. I have no idea what changing this does given how byond misbehaves sometimes.

Reply:
It sets the default FPS value for people who join.

As far as i know, if you've changed your FPS value already, it doesnt change it, you can also change it after its been set to this in game preferences without any additional effort.

Assuming that this just outright breaks shit for whatever reason, a hotfix would be extremely simple, but ideally this would make the game smoother.